_networkconfig/4_accounts.md

```diff
@@ -80,9 +80,13 @@ You need to link the PIV Authentication certificate to each of the user's accoun
 #### Enable User Name Hints
 You will want to enabling user name hints for your network domain.  This will modify the logon prompts for _Windows_ workstations and servers joined to the network domain.  Your users will be prompted to provide both the PIV credential PIN value and a User Name Hint value.
 
-Username Hint setting:
-
-- _Computer Configuration_ -> _Policies_-> _Administrative Templates_ -> _Windows Components_, and then expand _Smart Card_.
-- Select _Allow user name hint_
+#### Username Hint Setting:
```

Comment:
The sub-headings are actually linked above.  so we don't want to add another sub-heading without modifying the links above, or weaving it in...

i'll do a quick change...expect a commit against this pr